nodejs/src/eventhub.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  earliestEventPosition,
+  EventHubConsumerClient,
+  EventHubProducerClient,
+  SubscriptionEventHandlers
+} from "@azure/event-hubs";
+import { BlobCheckpointStore } from "@azure/eventhubs-checkpointstore-blob";
+import {
+  ContainerClient,
+  StorageSharedKeyCredential
+} from "@azure/storage-blob";
+import {
+  context as otContext,
+  SpanStatusCode,
+  trace
+} from "@opentelemetry/api";
+import { getTracer } from "./tracing";
+import { getEnvironmentVariable } from "./utils";
+
+// TODO: work out the naming conventions of these spans.
+const eventHubHandlers: SubscriptionEventHandlers = {
+  processEvents: async (events, context) => {
+    if (events.length === 0) {
+      return;
+    }
+
+    return getTracer().startActiveSpan(
```

Comment:
Doesn't JS EventHub SDK create their own spans for processing?

Reply:
It does create its own spans. What about parenting though? without this you'd end up with eventhub spans and storage-blob spans (for updateCheckpoint) - do app developers usually not group these under a single parent span?

Reply:
checkpointing should happen in the scope of SDK process span, isn't it?